Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -288,16 +186,11 @@ extension Datadog {
                     crashReportingPlugin: nil,
                     // While `.set(<feature>Endpoint:)` APIs are deprecated, the `datadogEndpoint` default must be `nil`,
                     // so we know the clear user's intent to override deprecated values.
-                    datadogEndpoint: nil,
+                    datadogEndpoint: .us1,
```

Comment:
Contrary to my previous comment, makes me wonder if it's not gonna be a source of support requests 🤔 

Maybe actually it's better to print a warning when it's not set - and make the developer consider what kind of value they want to use.

Saying that - we could see if it's possible to remove configuration and let the backend decide where to send stuff based on the API key.